osidb/tests/factories.py

```diff
@@ -126,24 +126,24 @@ class Params:
             # Flaw.FlawNistCvssValidation.REJECTED,
         ],
     )
-    summary = factory.LazyAttribute(
+    cve_description = factory.LazyAttribute(
         lambda f: "I am a spooky CVE"
         if f.is_mi
-        else ("random summary" if f.impact_requiring_summary else f.fallback)
+        else ("random cve_description" if f.impact_requiring_summary else f.fallback)
```

Comment:
`f.impact_requiring_summary` should be renamed to `f.impact_requiring_cve_description`. Similarly for `not_mi_with_summary` and  `not_mi_without_summary`